src/briefcase/integrations/flatpak.py

```diff
@@ -252,6 +252,7 @@ def run(self, bundle, app_name):
                     f"{bundle}.{app_name}",
                 ],
                 check=True,
+                stream_output=True,
```

Comment:
I can't build a Flatpak right now....still trying to figure out the underlying problem. Something with `libffi.so` not being available.

Reply:
Found the source of the issue:
- #858

Confirmed log is captured in the log.